docs/source/preferences/Preferences.rst

```diff
@@ -162,7 +162,7 @@ preferences helper as follows::
   class SplashScreenPreferences(PreferencesHelper):
       """ A preferences helper for the splash screen. """
 
-      PREFERENCES_PATH = 'acme.ui'
+      preferences_path = 'acme.ui'
```

Comment:
The use of `PREFERENCES_PATH` results in this warning message:
https://github.com/enthought/apptools/blob/4bf708f7b9e856c3fa4630c7de36821e31532ade/apptools/preferences/preferences_helper.py#L115